app/src/components/Registration/Dropdown.tsx

```diff
@@ -6,54 +6,59 @@ import {
     FormControl,
     Select,
 } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
 
-const DropdownSelect = (props: {
-    name: string
+const useStyles = makeStyles((theme) => ({
+    formControl: {
+        margin: theme.spacing(0),
+        minWidth: 200,
+    },
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+}))
+
+interface Props {
     value: string
     onChange: (event: object) => void
-    placeholder: string
     id: string
-    itemValue: string
-    className: string
-    htmlFor: string
     helperText: string
     options: Array<string>
```

Comment:
We may want to support a differing 'value' and 'text' property for the dropdown options. For example, our generic "select one" question types, the application should submit the ID of the response, which will be different from its display text